templates/simple/script.js

```diff
@@ -0,0 +1,15 @@
+function toggleScreenshot(e){
+    if(this.innerText === "Screenshot -"){
+        this.innerText = "Screenshot +";
+        this.nextElementSibling.style.display = "none";
+    } else {
+        this.innerText = "Screenshot -";
+        this.nextElementSibling.style.display = "block";
+    }
+}
+
+var toggleEls = document.querySelectorAll("a.toggle");
+
+for(var i = 0; i < toggleEls.length; i++){
+    toggleEls[i].addEventListener("click", toggleScreenshot);
+}
```

Comment:
Yeah, that was a bug in my change that i missed. The below commit fixes it. Im kinda in the middle of changes, so dont merge yet!